src/Utils/metaphysics.ts

```diff
@@ -1,3 +1,4 @@
+import "isomorphic-fetch"
```

Comment:
This was the first issue that popped up when starting down the SSR path but was an easy fix